src/test/java/com/xellitix/jenkins/plugintool/cli/command/HelpCommandTest.java

```diff
@@ -0,0 +1,29 @@
+package com.xellitix.jenkins.plugintool.cli.command;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class HelpCommandTest {
+
+    // Constants
+    private static final String NAME = "help";
+
+    // Fixtures
+    private HelpCommand command;
+
+    @Test
+    public void getName__Test() {
+        assertThat(command
```

Comment:
The indentation convention used for this project is:
indent: 2 spaces
continuation indent: 4 spaces

Please configure your IDE accordingly and auto-format this class.